helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/AvroRecordValidationIssue.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2022 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package com.linkedin.avroutil1.compatibility;
+
+import com.linkedin.avroutil1.compatibility.avropath.AvroPath;
+
+public class AvroRecordValidationIssue {
+    AvroPath path;
+    String msg;
+    boolean error; //otherwise warning
```

Comment:
This can be replaced with an enum to explicitly convey an error/warning.

Reply:
done